docs/docs/rest-api/public/api/v2/pods.raml

```diff
@@ -31,6 +31,8 @@ traits: !include ../traits.raml
     responses:
       201:
         description: Pod created successfully.
+        headers:
+          X-Marathon-Deployment-Id: string
```

Comment:
I missed that the RFC finally deprecated the X- prefix.
On Tue, Sep 6, 2016 at 6:35 PM James DeFelice <notifications@github.com>
wrote:

> In docs/docs/rest-api/public/api/v2/pods.raml
> <https://github.com/mesosphere/marathon/pull/4314#discussion_r77748176>:
>
> > @@ -31,6 +31,8 @@ traits: !include ../traits.raml
> >      responses:
> >        201:
> >          description: Pod created successfully.
> > +        headers:
> > +          X-Marathon-Deployment-Id: string
>
> dislike X-
>
> pretty sure that current recommendations are against using X- because it
> becomes somewhat meaningless over time if everyone uses X- for custom
> headers; people might as well use the non-X- form.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4314/files/b5f867166094bf52398e068dc3a70f2f7225febf#r77748176>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAweZncUwwFhnNm7OmDCpak0r9SUvN0mks5qnhT0gaJpZM4J2Y2N>
> .
>